iothub/device/src/InternalClient.cs

```diff
@@ -1225,26 +1225,26 @@ public async Task SetDesiredPropertyUpdateCallbackAsync(DesiredPropertyUpdateCal
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         public async Task SetDesiredPropertyUpdateCallbackAsync(DesiredPropertyUpdateCallback callback, object userContext, CancellationToken cancellationToken)
```

Comment:
Interesting choice of user passing `null` to this, instead of another method to clear the update callback. Passing null seems less obvious to me. Thoughts?

Reply:
We do this for Events already - DisableEventReceiveAsync(). Also we discussed with team on both approaches and this is the one we agreed upon. If we decide to do otherwise - we will have at least 4 new APIs unsubscribe for events, methods, properties, c2d. 

I agree it is not obvious and documenting this would be necessary here.

Reply:
Let's chat offline.

Reply:
Makes sense. I agree this is an acceptable approach, and prefer not to over-invest in track1.